src/unstable-temp/DownloadImageModal/ImageForm.tsx

```diff
@@ -46,6 +47,16 @@ const getDeviceTypeOptions = (deviceType: DeviceType) => {
 			});
 		}
 
+		// Add extra input field to advanced config
+		if (group.name === 'advanced') {
+			group.options.push({
+				message: t('info.provisioning_key_name'),
```

Comment:
```suggestion
				message: t('labels.provisioning_key_name'),
```
In order to have this also working outside the UI (eg balenaHub) you will also need to add a default translation for this in the `translationMap` found at:
See: https://github.com/balena-io-modules/rendition/blob/master/src/hooks/useTranslation.tsx


![image](https://user-images.githubusercontent.com/1295829/146016398-9f48126d-3e4b-40b5-8c75-10126584b044.png)